resources/schemas/dbscripts/postgresql/targetedms-19.23-19.24.sql

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ALTER TABLE targetedms.Replicate ALTER COLUMN Name TYPE VARCHAR(200);
```

Comment:
Is there a reason this doesn't have NOT NULL but the SQLServer variant does? Does Postgres retain that automatically but SQLServer doesn't when you do an ALTER COLUMN?

Reply:
Yes, that's what I saw on my dev machine.  Postgres retains the NOT NULL constraint but SQL Server does not.  I also saw this in a couple of other places in the code.  Example: exp-18.13-18.14.sql